src/main/java/com/aws/greengrass/clientdevices/auth/connectivity/ConnectivityInformation.java

```diff
@@ -79,8 +82,6 @@ public List<ConnectivityInfo> getConnectivityInfo() {
             if (getConnectivityInfoResponse.hasConnectivityInfo()) {
```

Comment:
connectivityInfoList is just going to be an empty list forever right?

Reply:
nvm, I see

Reply:
shouldn't unless im missing something, it'll have the info from CIS as before?